src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+
+function convertToNumber(currency) {
+  return Number(currency.split(',').join('.').slice(1));
+}
+
+thead.addEventListener('click', (e) => {
+  const title = e.target.textContent;
+  const index = e.target.cellIndex;
+
+  const rows = [...tbody.rows];
```

Comment:
This might be a performance issue. You are creating an unnecessary array of table rows. You can use the `tbody.rows` HTMLCollection directly in the `map` function on line 15. Remember, the `map` function can be used on any iterable object.